src/utils.rs

```diff
@@ -173,9 +195,9 @@ pub fn load_config() -> Env {
 
 fn set_oauth_tokens(json_response: &TraktAccessToken) {
     let mut config = Ini::new_cs();
-    let mut path = env::current_exe().unwrap();
-    path.pop();
-    path.push("credentials.ini");
+    let config_file = find_config_file();
+
+    let path = config_file.expect("Could not find credentials.ini");
```

Comment:
I assume that the user needs to move the credentials.ini file inside a manually created `distrakt` folder inside one of the config folders that are now documented in the readme? 

Would be nice if it was created automatically on the first run before closing. This way the need to create/download the file manually would disappear! And we could probably open the file explorer in said file's location!

Reply:
so they're not required to move it into the new discrakt folder - if there is a config dir in the currently documented location it'll use that, but will also check the standard config dir

I was thinking a `discrakt config` or `discrakt init` flow might be a nice idea, but thought I'd keep it small and not make any big sweeping changes :D